public/index.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Origin1</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href='https://fonts.googleapis.com/css?family=Architects+Daughter' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Arapey:400italic' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Libre+Baskerville:400italic' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Palanquin|Rancho' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Palanquin' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Rajdhani' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="vendors/styles/normalize.css" media="screen" charset="utf-8">
+    <link rel="stylesheet" href="styles/layout.css" media="screen" charset="utf-8">
+    <link rel="stylesheet" href="styles/home.css" media="screen" charset="utf-8">
+    <link rel="stylesheet" href="styles/contact.css" media="screen" charset="utf-8">
+    <link rel="stylesheet" href="styles/logo.css" media="screen" charset="utf-8">
+    <link rel="stylesheet" href="styles/coffeepage.css" media="screen" charset="utf-8">
+
+
+  <script id="coffeePageTemplate" type="text/x-handlebars-template">
```

Comment:
There's some basic indentation issues here in HTML. I would have liked to see those cleaned up during a final code review.